### PR TITLE
Add missing options

### PR DESCRIPTION
**Pull request by A6GibKm**

### CODE REVIEW

File: src/desktop/email.rs [hunk 1]
```diff
@@ -167,6 +168,13 @@ impl EmailRequest {
         self
     }
 
+    /// Sets the activation token.
+    #[must_use]
+    pub fn activation_token<'a>(mut self, activation_token: impl Into<Option<&'a str>>) -> Self {
```
Comment (bilelmoussaoui) [inline src/desktop/email.rs:173]: Do we expose that everywhere else as well?

**Comment by A6GibKm** [comment]
The print portal commit was marked as wip since this won't be exposed to the user, probably the `prepare_print` fn should be set to use a builder pattern otherwise it has too many optional arguments.

File: src/desktop/email.rs (hunk 1 above)
Comment (A6GibKm) [reply src/desktop/email.rs:173]: Nope.

Comment (bilelmoussaoui) [reply src/desktop/email.rs:173]: Then, we should for consistency or mark it as as private with a todo item saying we should handle it once we support passing activation

**Comment by bilelmoussaoui** [comment]
> The print portal commit was marked as wip since this won't be exposed to the user, probably the `prepare_print` fn should be set to use a builder pattern otherwise it has too many optional arguments.

I didn't spend time to make the print portal using builder pattern because it is too much.

**Comment by A6GibKm** [comment]
> > The print portal commit was marked as wip since this won't be exposed to the user, probably the `prepare_print` fn should be set to use a builder pattern otherwise it has too many optional arguments.
> 
> I didn't spend time to make the print portal using builder pattern because it is too much.

So just add a new argument?

**Comment by bilelmoussaoui** [comment]
> > > The print portal commit was marked as wip since this won't be exposed to the user, probably the `prepare_print` fn should be set to use a builder pattern otherwise it has too many optional arguments.
> > 
> > 
> > I didn't spend time to make the print portal using builder pattern because it is too much.
> 
> So just add a new argument?

For now yes

**Comment by bilelmoussaoui** [comment]
Could you drop the WIP from the last commit ? Otherwise lgtm

**Comment by A6GibKm** [comment]
https://github.com/flatpak/xdg-desktop-portal/pull/1094 hasn't been merged yet so...

**Pull request merged**